From 4ee78bfb8a526f250184958fda717032b47a5e96 Mon Sep 17 00:00:00 2001
From: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date: Tue, 30 Dec 2014 16:33:15 +0200
Subject: [PATCH] buildoutputparser: Respect $QTC_PREFIX.

Do what all other targets already do and take $$QTC_PREFIX into
consideration when setting the target's installation path.

Change-Id: I6ca87d36b345ec256518fe2d349e58d213f5c6d5
Reviewed-by: Christian Kandeler <christian.kandeler@theqtcompany.com>
Reviewed-by: Tobias Hunger <tobias.hunger@theqtcompany.com>


--- src/tools/buildoutputparser/buildoutputparser.pro
+++ src/tools/buildoutputparser/buildoutputparser.pro
@@ -13,7 +13,7 @@ include(../../rpath.pri)
 win32|equals(TEST, 1):DEFINES += HAS_MSVC_PARSER
 
 DESTDIR = $$IDE_BIN_PATH
-target.path = /bin
+target.path = $$QTC_PREFIX/bin
 INSTALLS += target
 
 SOURCES = \

